composer.json

```diff
@@ -1,19 +1,23 @@
 {
-    "name": "owncloud/files_external_s3",
-    "description": "ownCloud primary storage integration using S3 protocol",
-    "require": {
-        "guzzlehttp/guzzle": "^5.3",
-        "aws/aws-sdk-php": "^3.48"
-    },
-    "license": "GPL-2.0",
-    "config" : {
-        "optimize-autoloader": true,
-        "classmap-authoritative": false,
-        "platform": {
-            "php": "7.0.0"
-        }
-    },
-    "require-dev": {
-        "owncloud/coding-standard": "^1.0"
+  "name": "owncloud/files_external_s3",
+  "description": "ownCloud primary storage integration using S3 protocol",
+  "require": {
+    "dg/composer-cleaner": "^2.0",
+    "aws/aws-sdk-php": "^3.48"
+  },
+  "replace": {
+    "guzzlehttp/guzzle": "*"
```

Comment:
I googled this and now I understand. That's a neat thing, we should do this for all other external storages.

Reply:
any also for many other libs - e.g. symfony dependencies ...